### ISSUE TITLE
backdrop recommends to update even it's already last version

### ISSUE THREAD

**Issue by robertgarrigos** [backdrop/backdrop-issues#2698]
## Describe your issue or idea

I just updated to last 1.7.0 version. Still it says I need to update the core and it's recomending a previous version (1.7.0-preview of 5/2/17)

### Actual behavior (if reporting a bug)

See the screenshot:
[https://www.evernote.com/l/AATqZ3VT2D9L2qeRFzxuOUSBSsRtXEm_WQo](url)

**Comment by bd0bd** [comment]
BTW

![img_20170516_173229](https://cloud.githubusercontent.com/assets/13515302/26111509/f110a3d4-3a5d-11e7-87f0-da3da73ed690.JPG)

**Comment by klonos** [comment]
Seems that the regex that does the version comparison considers 1.7.0-whatever > 1.7.0

**Comment by bd0bd** [comment]
BD 1.6.3

Fatal error: Cannot redeclare class BackdropQueue in /core/modules/system/system.queue.inc on line 59

if I click ([Check manually](#)) at ` /admin/reports/updates/update`

**Comment by robertgarrigos** [comment]
the error is gone. I believe someone has fixed the xml history file which is responsible for that, as I've been reading in update module.

**Comment by quicksketch** [comment]
> I believe someone has fixed the xml history file which is responsible for that, as I've been reading in update module.

Yeah, I believe @jenlampton unpublished the `preview` release on backdropcms.org, thus fixing the XML feed for updates.

We still need to figure out the root of this in Update module though. Oddly we didn't seem to have any problems when `1.7.0-preview` was released. Existing `1.6.3` sites did not report an available update, which was correct (we don't want to recommend a preview over a final version). The problem with `1.7.0-preview` being recommended occurred *after* release of `1.7.0`.
